test/testsurround.c

```diff
@@ -198,6 +198,33 @@ main(int argc, char *argv[])
 
         SDL_CloseAudioDevice(dev);
     }
+    
+    char *deviceName;
+    SDL_AudioSpec spec;
+    
+    SDL_Log("Getting info from output device...");
+    if (SDL_GetDefaultAudioInfo(&deviceName, &spec, 0)) {
+        SDL_Log("Error when calling SDL_GetDefaultAudioInfo: %s\n", SDL_GetError());
+    } else {
+        SDL_Log("Info found!\n");
+        SDL_Log("Name: %s\n", deviceName);
+        SDL_Log("Sampling Rate: %d\n", spec.freq);
+        SDL_Log("Number of Channels: %d\n", spec.channels);
+        SDL_Log("Audio Format: %d\n", spec.format);
+        SDL_free(deviceName);
+    }
+    
+    SDL_Log("Getting info from input device...");
+    if (SDL_GetDefaultAudioInfo(&deviceName, &spec, 1)) {
+        SDL_Log("Error when calling SDL_GetDefaultAudioInfo: %s\n", SDL_GetError());
+    } else {
+        SDL_Log("Info found!\n");
```

Comment:
Minor change to the test logs, don't mind the diff the suggestion shows...
```suggestion
    if (SDL_GetDefaultAudioInfo(&deviceName, &spec, 0)) {
        SDL_Log("Error when calling SDL_GetDefaultAudioInfo: %s\n", SDL_GetError());
    } else {
        SDL_Log("Default audio output info found!\n");
        SDL_Log("Name: %s\n", deviceName);
        SDL_Log("Sampling Rate: %d\n", spec.freq);
        SDL_Log("Number of Channels: %d\n", spec.channels);
        SDL_Log("Audio Format: %d\n", spec.format);
        SDL_free(deviceName);
    }
    
    if (SDL_GetDefaultAudioInfo(&deviceName, &spec, 1)) {
        SDL_Log("Error when calling SDL_GetDefaultAudioInfo: %s\n", SDL_GetError());
    } else {
        SDL_Log("Default audio capture info found!\n");
```